WMIAdventure/frontend/src/js/components/main-menu/atoms/Option/Option.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import kuceBattle from "../../../../../assets/images/kuceBattle.png";
+import OptionLink from "./styled-components/OptionLink";
+import OptionHeader from "./styled-components/OptionHeader";
+import Span from "./styled-components/Span";
+import BattleImage from "./styled-components/BattleImage";
+import kucEditor from "../../../../../assets/images/kucEditor.svg";
+import EditorImage from "./styled-components/EditorImage";
+
+class Option extends React.Component {

```

Comment:
Może inaczej to nazwać? Mało zrozumiała ta nazwa, musiałem chwilę pomyśleć.
Może `MainMenuButton`?

Reply:
ok